src/main/java/iana/tasks/TaskList.java

```diff
@@ -0,0 +1,127 @@
+package iana.tasks;
+
+import iana.exception.IanaException;
+import iana.tasks.TaskList;
+import java.util.ArrayList;
+
+/**
+ * Represents the list that contains and manage all the current tasks.
+ */
+public class TaskList {
+    private ArrayList<Task> taskList;
+
+    /**
+     * Constructor for an empty task list.
+     */
+    public TaskList() {
+        this.taskList = new ArrayList<>();
+    }
+
+    /**
+     * Constructor for a task list given an ArrayList of Task.
+     * @param taskList ArrayList of task that contains current tasks.
+     */
+    public TaskList(ArrayList<Task> taskList) {
+        this.taskList = taskList;
+    }
+
+    /**
+     * Get the number of tasks in the task list.
+     * @return number of tasks.
+     */
+    public int size() {
+        return this.taskList.size();
+    }
+
+    /**
+     * Add a new task into the task list.
```

Comment:
> In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.)

Perhaps would you consider changing it to Adds a new task into the task list. Similarly for the `Return xxx` comments below.